liquibase-standard/src/test/java/liquibase/verify/saved_state/compareGeneratedSqlWithExpectedSqlForMinimalChangesets/createView/edb.sql

```diff
@@ -1,4 +1,4 @@
 -- Database: edb
 -- Change Parameter: selectQuery=SELECT id, name FROM person WHERE id > 10
 -- Change Parameter: viewName=v_person
-CREATE VIEW v_person AS SELECT id, name FROM person WHERE id > 10;
+CREATE VIEW v_person AS SELECT id, name FROM person WHERE id > 10;
```

Comment:
Please undo this change.

Reply:
there is no change there just GH getting mad

Reply:
I just pushed up the change to undo this.